doc/apps/ciphers.pod

```diff
@@ -657,6 +661,16 @@ Note: these ciphers can also be used in SSL v3.
  DHE_PSK_WITH_AES_128_CCM_8                DHE-PSK-AES128-CCM8
  DHE_PSK_WITH_AES_256_CCM_8                DHE-PSK-AES256-CCM8
 
+=head2 ChaCha20-Poly1305 cipher suites from draft-ietf-tls-chacha20-poly1305-04, extending TLS v1.2
```

Comment:
FWIW, if you remove the numbers completely, the ietf website redirects to the latest version https://tools.ietf.org/html/draft-ietf-tls-chacha20-poly1305

So the reference won't get out-of-date so quickly.